e2e/local/util.go

```diff
@@ -48,3 +53,10 @@ func StopDockerContainers() {
 		Docker(args...)
 	}
 }
+
+func kamelWithContextAndMaybeStaging(ctx context.Context, args ...string) *cobra.Command {
```

Comment:
It looks a bit verbose to me. Maybe we can just name it `kamelWithContext` here.